### DISCUSSION TITLE
Invite to share feedback!

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-inQuiry/Community#3] [Announcements]
Folks - the entire inQuiry building block team has been spending a significant amount of time documenting the functional and technical capabilities. The result of this ongoing exercise is our new microsite - https://inquiry.sunbird.org/. While the microsite is still WIP, we would like the community to review our new microsite, share feedback and engage with us so that we can collaborate and make the microsite useful for all adopters of Sunbird inQuiry.  You can route your suggestions/feedback/comments @ https://github.com/Sunbird-inQuiry/Community/discussions.

@surendrasinghs @pallakartheekreddy @krgauraw @Nadiya496 @rajnishdargan @itsvick @SanthoshVasabhaktula @vrayulu @ashokreddy1208 @kameshbhr @mmanimegalai @luckynvdu @reshmi-nair @sharathkashyap @gandham-santhosh @Prerna789
@chitranshu-keshav @coolbung @aayushkedia95 @ChakshuGautam @rhwarrier @alok-os @vinukumar-vs @jagadishbabu @madhucr @coolbung @vijiurs @aks30 @kiranharidas187

**Comment by pramodkvarma** [comment]
Wonderfully done! I went through it. Few comments:

1. Btw, link to telemetry is pointing straight to a specific version on Github. Shouldn't it point to [telemetry.sunbird.org](https://telemetry.sunbird.org)? 
2. Also, I see source code link on left menu (under ENGAGE section) is missing. Is this because source code is not yet isolated? When is that happening?
3. I noticed that source code for Question/set micro-service is pointing to whole Knowledge platform!! Don't we have a separate building block called [Knowlg](https://knowlg.sunbird.org/). How's that separated out. It's confusing. I am hoping this will get fixed.

Please create issue tracker items on this community repo to track and close these. NOT internal JIRA please. @Krishnaj20 

Again, excellent job! Kudos to everyone!

**Comment by Krishnaj20** [comment]
Thank you @pramodkvarma for the feedback! Quick update on your comments:
- the telemetry url is now corrected in the microsite.
- the ENGAGE section is designed to have the link to the discussion forums. Pls clarify if you are referring to source code in this section or some other section.
- Separating out question & question set micro-service from Knowlg is planned in the AMJ quarter. 

We will reach out to you to close a few questions about the issue tracker. Till then request you to post any documentation related issues [here](https://github.com/Sunbird-inQuiry/Community/discussions/categories/documentation-issues).
